presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++                                     +                                          +                                            +not to wait for chunks of data to start processing  +
++                                     +                                          +                                            +them while scanning tables. This may cause faster   +
++                                     +                                          +                                            +processing with lower latency and downtime but some +
++                                     +                                          +                                            +operators may do things more efficiently when       +
++                                     +                                          +                                            +working with chunks of data.                        +
++-------------------------------------+------------------------------------------+--------------------------------------------+----------------------------------------------------+
++       ``resource_overcommit``       +               ``Boolean``                +                 ``false``                  +Use resources which are not guaranteed to be        +
++                                     +                                          +                                            +available to the query. By setting this property    +
++                                     +                                          +                                            +you allow to exceed limits of memory available per  +
++                                     +                                          +                                            +query processing and session. This may cause        +
++                                     +                                          +                                            +resources to be used more efficiently allowing to   +
++                                     +                                          +                                            +perform larger queries but may cause some           +
++                                     +                                          +                                            +indeterministic query drops due to lacking memory   +
++                                     +                                          +                                            +on cluster or single workers.                       +
++-------------------------------------+------------------------------------------+--------------------------------------------+----------------------------------------------------+
++``plan_with_table_node_partitioning``+               ``Boolean``                +                  ``true``                  +**Experimental.** Adapt plan to pre-partitioned     +
```

Comment:
I'm not sure if I understand that ;-)
I'll try to elaborate more and provide example.
To be done.